airflow/contrib/hooks/openfaas_hook.py

```diff
@@ -26,7 +26,7 @@
 
 class OpenFaasHook(BaseHook):
     """
-    Interact with Openfaas to query, deploy, invoke and update function
+    Interact with OpenfaaS to query, deploy, invoke and update function
```

Comment:
```suggestion
    Interact with OpenFaaS to query, deploy, invoke and update function
```
isn't it? https://github.com/openfaas/faas#openfaas----serverless-functions-made-simple

Reply:
True.